modules/core/client/views/footer.client.view.html

```diff
@@ -0,0 +1,10 @@
+<div class="container" ng-controller="FooterController as vm">
+  <div class="row">
+    <div class="col-md-12">
+      <hr/>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-md-4 col-md-offset-4">&copy; {{vm.year}} MEAN.js | All rights reserved</div>
```

Comment:
@jloveland Agreed as should the header.  How would you go about getting the server config to the client?  I think this opens up a larger can of worms that we need to discuss.